### ISSUE TITLE
192.***.***.15: bridge request 1: get /config 192.***.***.15: bridge http status 404 Not Found

### ISSUE THREAD

**Issue by erikarendsen** [ebaauw/homebridge-hue#336]
#### Issue

The port of my deCONZ change to 8080 in stead of 80. Since then I've got an error. Can I change the portnumber in the homebridge-hue settings anywhere? Or is there another solution?

#### Log Messages

```
[2018-4-30 11:58:41] [Hue] 192.***.***.15: bridge request 1: get /config
[2018-4-30 11:58:41] [Hue] 192.***.***.15: bridge http status 404 Not Found
[2018-4-30 11:58:41] [Hue] Hue: Philips BSB002 bridge v1802201122, api v1.24.0
[2018-4-30 11:58:41] [Hue] Hue: 1 accessories
[2018-4-30 11:58:56] [Hue] 192.***.***.15: bridge request 2: get /config
[2018-4-30 11:58:56] [Hue] 192.***.***.15: bridge http status 404 Not Found
```

#### Debug Files

[2018-4-30 12:12:08] [Hue] npm registry: get /homebridge-hue ok
[2018-4-30 12:12:08] [Hue] config.json: {"sensors":false,"platform":"Hue","name":"Hue","nativeHomeKitSensors":false,"lights":false,"users":{"001788FFFE4973C6":"***************************************1"}}
[2018-4-30 12:12:08] [Hue] meethue portal: get /api/nupnp
[2018-4-30 12:12:08] [Hue] dresden appspot ipv4: get /discover
[2018-4-30 12:12:08] [Hue] dresden appspot ipv6: get /discover
[2018-4-30 12:12:08] [Hue] upnp search: searching at 239.255.255.250:1900 for 5 seconds
[2018-4-30 12:12:08] [Hue] upnp search: found bridge 00212EFFFF0178BA at 192.***.***.15
[2018-4-30 12:12:08] [Hue] meethue portal: get /api/nupnp ok
[2018-4-30 12:12:08] [Hue] meethue portal: found bridge 001788FFFE4973C6 at 192.***.***.11
[2018-4-30 12:12:08] [Hue] dresden appspot ipv6: get /discover ok
[2018-4-30 12:12:08] [Hue] dresden appspot ipv6: found gateway 00212EFFFF0178BA at 192.***.***.15
[2018-4-30 12:12:08] [Hue] dresden appspot ipv4: get /discover ok
[2018-4-30 12:12:10] [Hue] upnp search: found bridge 001788FFFE4973C6 at 192.***.***.11
[2018-4-30 12:12:13] [Hue] upnp search: done
[2018-4-30 12:12:13] [Hue] 192.***.***.15: bridge request 1: get /config
[2018-4-30 12:12:13] [Hue] 192.***.***.11: bridge request 1: get /config
[2018-4-30 12:12:13] [Hue] 192.***.***.15: bridge request 1: get /config
[2018-4-30 12:12:13] [Hue] 192.***.***.15: bridge http status 404 Not Found
[2018-4-30 12:12:13] [Hue] 192.***.***.11: bridge request 1: ok
[2018-4-30 12:12:13] [Hue] Hue: Philips BSB002 bridge v1802201122, api v1.24.0
[2018-4-30 12:12:13] [Hue] Hue: bridge request 2: get /
[2018-4-30 12:12:13] [Hue] Hue: bridge request 2: ok
[2018-4-30 12:12:13] [Hue] Hue: bridge request 3: get /groups/0
[2018-4-30 12:12:13] [Hue] Hue: bridge request 3: ok
[2018-4-30 12:12:13] [Hue] Hue: 0 sensors
[2018-4-30 12:12:13] [Hue] Hue: 0 lights
[2018-4-30 12:12:13] [Hue] Hue: 0 schedules
[2018-4-30 12:12:13] [Hue] Hue: 0 rules
[2018-4-30 12:12:13] [Hue] Hue: 0 groups
[2018-4-30 12:12:13] [Hue] Hue: 1 accessories
[2018-4-30 12:12:28] [Hue] 192.***.***.15: bridge request 2: get /config
[2018-4-30 12:12:28] [Hue] 192.***.***.15: bridge request 2: get /config
[2018-4-30 12:12:28] [Hue] 192.***.***.15: bridge http status 404 Not Found

**Comment by ebaauw** [comment]
Looking at the code, it would seem that I don’t check for the port.  Odd, I would have sworn I did... maybe in another branch.  Will have to check tonight or coming weekend.

**Comment by erikarendsen** [comment]
Thx! Adding the hosts in the config.json is the solution. Keep up the good work!

**Comment by ebaauw** [comment]
Fixed in v0.9.3.

**Comment by QHose** [comment]
Nice! this also solved this issue ` [2019-3-7 08:26:07] [Hue] 192.***.***.115: bridge request 18: get /config
[2019-3-7 08:26:07] [Hue] 192.***.***.115: bridge http status 404 Not Found`
